reports/jsonld-gold-earl.ttl

```diff
@@ -0,0 +1,18485 @@
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<https://github.com/piprate/json-gold/tree/v0.3.0> a doap:Version;
+  doap:created "2020-04-06"^^xsd:date;
+  doap:name "json-gold-v0.3.0";
+  doap:revision "v0.3.0" .
+
+<https://github.com/piprate/json-gold> a <http://www.w3.org/ns/earl#TestSubject>,
+    doap:Project,
+    <http://www.w3.org/ns/earl#Software>;
+  dc:title "JSON-goLD";
+  dc:creator <https://github.com/kazarena>;
+  dc:date "2020-04-06"^^xsd:date;
+  doap:description "A JSON-LD processor for Go";
+  doap:developer <https://github.com/kazarena>;
+  doap:homepage <https://github.com/piprate/json-gold>;
+  doap:license <https://github.com/piprate/json-gold/blob/master/LICENSE>;
+  doap:name "JSON-goLD";
+  doap:programming-language "Go";
+  doap:release <https://github.com/piprate/json-gold/tree/v0.3.0> .
+
+<https://github.com/kazarena> a <http://www.w3.org/ns/earl#Assertor>,
+    foaf:Person;
+  foaf:homepage <https://github.com/kazarena>;
+  foaf:name "Stan Nazarenko" .
+
+ [
+    a <http://www.w3.org/ns/earl#Assertion>;
+    <http://www.w3.org/ns/earl#assertedBy> <https://github.com/kazarena>;
+    <http://www.w3.org/ns/earl#mode> <http://www.w3.org/ns/earl#automatic>;
+    <http://www.w3.org/ns/earl#result> [
+      a <http://www.w3.org/ns/earl#TestResult>;
+      dc:date "2020-04-06T17:15:22.676634"^^xsd:date;
```

Comment:
@gkellogg Shouldn't these be xsd:dateTime?  Or if it's xsd:date then truncated to just yyyy-mm-dd?

Reply:
The range of `dc:date` is `rdfs:Literal`, so it doesn't really matter from that perspective, and it's not used in the report generation in any case.